src/components/dialogs/create-study-dialog/importParametersSection.tsx

```diff
@@ -10,18 +10,36 @@ import { Divider } from '@mui/material';
 import { FlatParameters } from '@gridsuite/commons-ui';
 import React, { useState, FunctionComponent } from 'react';
 import AdvancedParameterButton from './advancedParameterButton';
+import { CURRENT_PARAMETERS } from '../../utils/field-constants';
+import { useFormContext } from 'react-hook-form';
 
 interface ImportParametersSectionProps {
-    onChange: () => void;
-    currentParameters: Record<string, any>;
     formatWithParameters: any[]; // You can replace `any` with a more specific type if available.
 }
 
 const ImportParametersSection: FunctionComponent<
     ImportParametersSectionProps
-> = ({ onChange, currentParameters, formatWithParameters }) => {
+> = ({ formatWithParameters }) => {
     const [isParamsDisplayed, setIsParamsDisplayed] = useState<boolean>(false);
 
+    const { setValue, watch } = useFormContext();
+
+    const currentParameters = watch(CURRENT_PARAMETERS);
+
+    const handleParamsChange = (
+        paramName: string,
+        value: string,
+        isEdit: boolean
+    ): void => {
+        if (!isEdit) {
+            setValue(CURRENT_PARAMETERS, {
+                ...currentParameters,
```

Comment:
As I told you, please remove the watch and use getValue here instead. It avoids unnecessary rerenders.